pages/auth/signup.tsx

```diff
@@ -183,22 +198,64 @@ const SignupPage: React.FC<{}> = () => {
                         label="Let's partner up"
                         description="Let's level up your digital profile, together."
                     />
-                    {!showEmailForm && <>
-                        <SocialAuthComponent />
-                        <div>
-                            <Button
-                                onClick={e => {
-                                    e.preventDefault()
-                                    setShowEmailForm(true)
-                                }}
-                                type={ButtonType.SECONDARY}
-                                label='Signup with Email'
-                            />
-                        </div>
-                    </>}
-
+                    {!showEmailForm && (
+                        <>
+                            <SocialAuthComponent />
+                            <div>
+                                <Button
+                                    onClick={(e) => {
+                                        e.preventDefault();
+                                        setShowEmailForm(true);
+                                    }}
+                                    type={ButtonType.SECONDARY}
+                                    label='Signup with Email'
+                                />
+                            </div>
+                        </>
+                    )}
+
+                    {loading && (
```

Comment:
@ibrahimkholil6210 instead of doing it manually at each component instead can be done at a common hoc level. 
please find backdrop component and update in there.

Reply:
@sifatul I don't want to show it inside any backdrop. As a user when i see a backdrop it feels like the app is not responsive and takes my attention away from the app. But this kind of loading in a particular section used will feel like part is doing something he/she might look around the other part of the app in the mean time we can hope loading will finish thus provide much better UX then showing backdrop. 

Please let me know if you have other POV regarding this.

Reply:
@ibrahimkholil6210 i agree with you completely. I did not suggest to use backdrop either. 
FYI this concept is nice. 
1. but for the sake of best practice  need a global component to make sure that developers don't have write it in every component manually > so replace the code of `components/common/Backdrop.tsx` with your this animation
 2. this kind of paper plan flying can not be shown for a section or sidebar , image a container of width 400px and height of 400px, during loading showing this animation might not be the best thing to do. 
 
Would love to see how others think